html_css/js/slider.js

```diff
@@ -0,0 +1,138 @@
+export default class Slider {
+
+  constructor(selector, data) {
+    this.selector = selector;
+    this.data = data;
+  }
+
+  initSlider() {
+    this.slider = document.querySelector(this.selector);
+    this.processDataForSlides(this.data);
+    this.initIventListener();
+    this.page = 0;
```

Comment:
Could be just class property. And in general better to define class properties before

Reply:
Transferred to Constructor